src/components/v5/frame/ColonyHome/partials/ReputationChart/consts.ts

```diff
@@ -16,5 +22,15 @@ export const summaryLegendColor = {
   '--color-teams-indigo-500': 'bg-teams-indigo-500',
   '--color-teams-grey-500': 'bg-teams-grey-500',
   '--color-teams-grey-100': 'bg-gray-100',
-  default: '--color-blue-400',
+  '--color-gray-400': 'bg-gray-400',
+  '--color-gray-200': 'bg-gray-200',
+  default: 'bg-blue-400',
+};
+
+export const pieChartConfig: Partial<PieSvgProps<ReputationChartDataItem>> = {
+  colors: ({ data }) => `var(${data.color})`,
```

Comment:
Awesome 👏 I will make use of this as well

Reply:
Though, I think on the theme change between light/dark we need to refresh the colors 😢 


https://github.com/user-attachments/assets/ae5c6641-9faf-404e-9d3a-c69ddecc2d79

Reply:
How about we make use of a hook to update the chart config, so that the colors properly refresh upon the theme changing?

`hooks.ts`
```
import { useEffect, useState } from 'react';

import { pieChartConfig } from './consts';

export const useGetPieChartConfig = () => {
  const [config, setConfig] = useState(pieChartConfig);

  useEffect(() => {
    const updateConfig = () => {
        setConfig(pieChartConfig);
    };
    const observer = new MutationObserver(updateConfig);

    observer.observe(document.documentElement, {
      attributes: true,
      attributeFilter: ['style'],
    });

    updateConfig();

    return () => {
      observer.disconnect();
    };
  }, []);

  return config;
};

```

Reply:
Actually seems the colors properly update even without this hook 🤔 so nevermind